Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM golang:1.13 as modules
+ADD ./go.mod /m/
+RUN cd /m && go mod download
+FROM golang:1.13 as builder
+
+RUN mkdir -p /opt/resource/
+
+COPY --from=modules /go/pkg/ /go/pkg/
+
+WORKDIR /opt/resource/
+COPY cmd             cmd
+COPY config          config
+COPY middleware      middleware
+COPY model           model
+COPY server          server
+COPY storage         storage
+COPY go.*            ./
```

Comment:
Is it reasonable to add each package to the container instead of entire project?

Reply:
Agree with you. Changed it

Reply:
And now we will include everything to docker container?

Reply:
Add .dockerignore whith list of exceptions